core/src/raw/adapters/kv/backend.rs

```diff
@@ -230,6 +230,58 @@ impl<S: Adapter> Accessor for Backend<S> {
 
         Ok((RpList::default(), pager))
     }
+
+    async fn rename(&self, from: &str, to: &str, _: OpRename) -> Result<RpRename> {
+        match self.copy(from, to, OpCopy {}).await {
```

Comment:
I prefer to call `get`/`set` directly instead.

Reply:
My initial version directly called it, but this will generate many redundant codes.

Reply:
> My initial version directly called it, but this will generate many redundant codes.

That's OK. OpenDAL's prefer clean and esay understand code.

Reply:
fixed.